Userland/Libraries/LibVideo/VP9/Parser.cpp

```diff
@@ -12,6 +12,8 @@
 
 namespace Video::VP9 {
 
+#define TRY_READ(expression) DECODER_TRY(DecoderErrorCategory::Corrupted, expression)
```

Comment:
Why is this not defined in the `DecoderError.h` file?

Reply:
Because this is only relevant during the parsing process, so it seemed pointless to make it accessible to other files. I have a few Category::Corrupted errors in Decoder.cpp, but not ones that would use the TRY syntax, so it wasn't applicable there. This macro is just for convenience to make the hundreds of try calls less verbose in Parser.

Should I move it anyway?

Reply:
I'll leave this as is and resolve this conversation.